.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Issue Template
+about: Issue Template
+
+---
+
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
+v                            ✰  Thanks for opening an issue! ✰    
+v    Before smashing the submit button please review the template.
+v   Please also ensure that this is not a duplicate issue. 
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  --> 
+
+## Summary
+<!-- Concisely describe the issue --> 
+
+## Problem definition / steps to reproduce
+<!-- 
+For Bugs: What commands in order should someone run to reproduce your problem
+For Proposals: Outline the problem which merits 
+--> 
+
+## Proposed solution 
+<!-- 
+For Bugs: (if applicable) proposed steps for solution
+For Proposals: The Proposal
```

Comment:
I think it may confuse users if they think they have to propose a solution for bugs in order to file the issue. B/c of that, I think we should tag this as proposals only. 

Also, should we add a note saying that proposals will be closed unless there is an actual concrete proposal? (I.e. for large feature requests, there has to be a description of what needs to be implemented, for small feature requests where its a (relatively) simple change this isn't needed)

Reply:
I'm going to separate this out into multiple templates to distinguish bug fixes vs. feature requests - should alleviate some of the comments.

Reply:
Awesome!

Reply:
Added this to the proposal template `Word of Caution: poorly thought-out proposals may be rejected without deliberation`

Reply:
Addressed comments